lib/BloqadeLattices/src/bounded_lattice.jl

```diff
@@ -77,13 +82,11 @@ Returns the dimensions of the `BoundedLattice` (e.g.: `2` for 2D, `3` for 3D)
 
 ```jldoctest; setup=:(using BloqadeLattices)
 julia> bl = parallelepiped_region(ChainLattice(),(4,);pbc=true) # create a 1D BoundedLattice
-BoundedLattice{ChainLattice, Parallelepiped{1, Float64}}(ChainLattice(), Parallelepiped{1, Float64}([4.0;;], [0.25;;]), [(0.0,), (1.0,), (2.0,), (3.0,)], true)
```

Comment:
If the output of the REPL is omitted in the example use case, a semicolon should be suffixed to indicate there is no output on execution.